tests/data/almanack/entropy/add_data.py

```diff
@@ -51,36 +17,42 @@ def commit_changes(directory: str, message: str):
         directory (str): The directory containing the Git repository.
         message (str): The commit message.
     """
-    subprocess.run(["git", "add", "."], check=True, cwd=directory)
-    subprocess.run(["git", "commit", "-m", message], check=True, cwd=directory)
+    repo = git.Repo(directory)
+    repo.git.add(".")
+    repo.index.commit(message)
+
 
+def add_data():
```

Comment:
Consider using a different name for this function or the module (filename) to help keep these distinct.